### PR TITLE
1810BE - Bradley Niedt - Museo

### PR DESCRIPTION
**Pull request by Bradniedt**
Here is my submission for Museo - got almost halfway through iteration 4.

### CODE REVIEW

File: lib/curator.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+class Curator
+  attr_reader :artists,
+              :photographs
+  def initialize
+    @artists = []
+    @photographs = []
+  end
+
+  def add_photograph(photo)
+    @photographs << photo
```
Comment (memcmahon) [inline lib/curator.rb]: Watch the spec!  You should be creating the Photo and Artist objects in these add methods, _not_ in your tests

File: lib/curator.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    end
+  end
+
+  def find_photographs_by_artist(artist)
+    @photographs.find_all do |photograph|
+      photograph.artist_id == artist.id
+    end
+  end
+
+  def artists_with_multiple_photographs
```
Comment (memcmahon) [inline lib/curator.rb]: Don't forget that you can re-use methods as helpers!  You could do something like: 

```
@artists.find_all do |artist|
  find_photographs_by_artist(artist).count > 1
end
```

File: lib/curator.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      find_photographs_by_artist(artist)
+    end
+    artists_work.keep_if do |photos|
+      photos.length > 1
+    end.flatten!
+    id = artists_work[0].artist_id
+    artists << find_artist_by_id(id)
+  end
+
+  def photographs_taken_by_artists_from(country)
```
Comment (memcmahon) [inline lib/curator.rb]: GREAT!

**Comment by Bradniedt** [comment]
Ohhhhhhh- I  should have reached out to clarify- I thought we wanted to
create the item with those attributes then just add them to the
photos/artist collection. That makes so much more sense that the add_
methods were object builders!

Thanks!

**Pull request closed without merging**